codex-rs/core/src/client.rs

```diff
@@ -922,6 +925,10 @@ fn try_parse_retry_after(err: &Error) -> Option<Duration> {
     None
 }
 
+fn is_context_window_error(error: &Error) -> bool {
+    error.code.as_deref() == Some("context_length_exceeded")
```

Comment:
Where does this string come from? Is there some `const` we should be referencing here instead?